tests/unit/test_shodan.py

```
@@ -0,0 +1,126 @@
+        self.assertEqual(CONFIG.addons.shodan.api_key, '')
+
+
+def _load_fixture():
+    path = os.path.join(os.path.dirname(__file__), '..', 'fixtures', 'shodan_output.json')
+    with open(path) as f:
+        return json.load(f)
+
+
+class TestShodanMapping(unittest.TestCase):
+    def _run(self):
+        from secator.output_types import Ip, Subdomain, Port, Technology, Vulnerability, Tag
```

Flake8 (7.3.0)

[error] 'secator.output_types.Ip' imported but unused

(F401)